.cargo/config.toml

```diff
@@ -0,0 +1,5 @@
+[target.'cfg(target_os = "windows")']
+rustflags = ["-Clink-args=/guard:cf /DYNAMICBASE /CETCOMPAT"]

```

Comment:
Isn't `/guard:cf /DYNAMICBASE /CETCOMPAT` msvc only?

Reply:
those flags are respected by the linker that Rust invokes, such that when passed on Windows they have the desired effect. Rust either calls `lld-link.exe` or `link.exe` which both support these flags.